packages/base/src/commonMain/kotlin/elide/struct/codec/TreeSetCodec.kt

```diff
@@ -14,27 +14,24 @@
 package elide.struct.codec
 
 import kotlinx.serialization.KSerializer
-import kotlinx.serialization.builtins.MapSerializer
+import kotlinx.serialization.builtins.SetSerializer
 import kotlinx.serialization.descriptors.SerialDescriptor
 import kotlinx.serialization.encoding.Decoder
 import kotlinx.serialization.encoding.Encoder
-import elide.struct.MutableTreeMap
+import elide.struct.TreeSet
 
-/**
- *
- */
```

Comment:
nit: doc, but that's on me, don't let it block merge

Reply:
no that empty doc was removed, I added a simple docstring.

Reply:
Ah yes, thank you for correcting me